scripts/ui_release.sh

```diff
@@ -10,8 +10,10 @@ cd ui/
 files=("../LICENSE" "../CHANGELOG.md")
 workspaces=$(jq -r '.workspaces[]' < package.json)
 publish_workspaces=$(for workspace in ${workspaces}; do 
-  # package "app" is private so we shouldn't try to publish it.
-  if [[ "${workspace}" != "app" ]] && [[ "${workspace}" != "e2e" ]]; then
+  # packages are private so we shouldn't try to publish them.
+  # TODO: see if we can do something smarter here by checking for "private" set
+  # to true in a given package's package.json.
```

Comment:
Part of me is really tempted to do this and rewrite this as a node script because it would be a lot easier for UI folks to maintain than bash, but I'm not digging into it as part of this initial effort.